File: examples/aaa/fizz/buzz.c
#include <stdio.h>

int main(void) {
    for (int i = 1;; ++i) {
        if (i % 15 == 0) {
            puts("fizz buzz!");
        } else if (i % 3 == 0) {
            puts("fizz...");
        } else if (i % 5 == 0) {
            puts("buzz?");
        } else {
            printf("%d\n", i);
        }
    }
}

File: examples/aaa/fizz/buzz.hs
fizzBuzz n = case (n `mod` 3, n `mod` 5) of
  (0, 0) -> "fizz buzz!"
  (0, _) -> "fizz..."
  (_, 0) -> "buzz?"
  (_, _) -> show n

main = mapM_ (putStrLn . fizzBuzz) [1..]
